Assets/Scripts/Raycaster.cs

```diff
@@ -0,0 +1,21 @@
+﻿using UnityEngine;
```

Comment:
I don't really think this class needs to exist, just use a raycaster in the classes where it is needed instead of having a separate component for it.